frontend/src/components/ResultRoutes/ResultRoutes.tsx

```diff
@@ -168,8 +168,8 @@ const ResultRoutes: React.FC<ResultRoutesProps> = ({ origin, destination }) => {
   }, [activeSlideIndex]);
 
   return (
-    <>
-      <Header />
+    <IonPage id="main-content">
```

Comment:
Why did you remove this? Removing the id is okay, but I think IonPage Directive is needed for proper navigation.